eng/common/pipelines/templates/steps/policheck.yml

```diff
@@ -0,0 +1,36 @@
+
+parameters:
+  ExclusionDataBaseFileName: ''
+  TargetDirectory: ''
+  PublishAnalysisLogs: false
+  PoliCheckBlobSAS: '$(azuresdk-policheck-blob-SAS)'
+
+steps:
+  - pwsh: |
+      azcopy copy "https://azuresdkartifacts.blob.core.windows.net/policheck/${{ parameters.ExclusionDataBaseFileName }}.mdb?${{ parameters.PoliCheckBlobSAS }}" `
+      "$(Build.BinariesDirectory)"
+    displayName: 'Download PoliCheck Exclusion Database'
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
+    displayName: 'Run PoliCheck'
+    inputs:
+      targetType: F
+      targetArgument: '$(Build.SourcesDirectory)/${{ parameters.TargetDirectory }}'
```

Comment:
When we start going to enable this in PR validations will this correctly work?

Reply:
Yes this is a way to scope the run to the package triggering the PR. It should not be a problem.